front/src/modules/ui/title/components/__stories__/MainSectionTitle.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { ThemeProvider } from '@emotion/react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { CatalogDecorator } from '~/testing/decorators/CatalogDecorator';
+import { ComponentDecorator } from '~/testing/decorators/ComponentDecorator';
+
+import { MainSectionTitle } from '../MainSectionTitle';
+
+import { theme1, theme2, theme3 } from './themes';
+
+const meta: Meta<typeof MainSectionTitle> = {
+  title: 'UI/Title/MainSectionTitle',
+  component: MainSectionTitle,
+  decorators: [ComponentDecorator],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof MainSectionTitle>;
+
+const defaultArgTypes = { control: false };
+
+const args = {
+  children: 'Lorem ipsum',
+};
+
+export const Default: Story = {
+  args,
+  decorators: [ComponentDecorator],
+};
+
+export const Catalog: Story = {
+  args,
+  argTypes: {
+    children: defaultArgTypes,
+  },
+  parameters: {
+    providers: Array(4).fill(ThemeProvider),
```

Comment:
we should not do that, storybook is already aware of lightTheme and darkTheme that you can switch using storybook top bar